pages/search.js

```diff
@@ -89,7 +89,7 @@ export default function Search({ users }) {
       <Page>
         <h1 className="text-4xl mb-4 font-bold">Search</h1>
         <input
-          placeholder="Search users"
+          placeholder="Search users, tags or language"
```

Comment:
Maybe make it languages (plural)?

Reply:
Hey, I have made the changes, and thanks for the PR thing I didn't know about it